src/components/leaderboards/List.vue

```diff
@@ -73,8 +89,18 @@ function filterChanged() {
 			</form>
 		</div>
 
-		<div v-if="searchedQuery" class="results-text" >
-			Displaying {{ boards.data.length }} of {{ boards.total }} results for "{{ searchedQuery }}" ({{ searchedStatus }}).&nbsp;
+		<div style="align-self: flex-end">
+			<Paginator
+				:total="boards.total"
+				:limit="limit ?? boards.limitDefault"
+				:page="page"
+			/>
+		</div>
+
+		<div v-if="searchedQuery" class="results-text">
+			Displaying {{ boards.data.length }} of {{ boards.total }} results for "{{
+				searchedQuery
```

Comment:
do you have the formatter running, this looks off.

Reply:
Lemme try running it again